templates/controllers/modals/submissionMetadata/form/issueEntrySubmissionReviewForm.tpl

```diff
@@ -54,3 +54,13 @@
 	{/if}
 
 </form>
+
+{if $formParams.revision && ($formParams.revision < $latestRevisionId)}
```

Comment:
Javascript should be written in external controllers, not embedded in templates.